### PR TITLE
Adding the ResizeObserver polyfill

### PR DESCRIPTION
**Pull request by andrewgadziksonos**
Adding the ResizeObserver polyfill - Feature Request https://github.com/Financial-Times/polyfill-library/issues/211

### CODE REVIEW

File: polyfills/ResizeObserver/config.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		"android": "<67",
+		"firefox_mob": "*",
+		"ie_mob": "*",
+		"op_mob": "*",
+		"ios_saf": "*"
+	},
+	"repo": "https://github.com/que-etc/resize-observer-polyfill",
+	"docs": "https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver",
+	"spec": "https://drafts.csswg.org/resize-observer-1",
+	"license": "W3C",
```
Comment (JakeChampion) [inline polyfills/ResizeObserver/config.json]: @que-etc has the polyfill licensed as [MIT](https://github.com/que-etc/resize-observer-polyfill/blob/master/LICENSE)
```suggestion
	"license": "MIT",
```

**Comment by andrewgadziksonos** [comment]
Opps I missed that from the polyfill implementation.  Thank you for catching that

**Comment by aronwoost** [comment]
This is an awesome PR and I'd love to see it happen soon. ❤️😘

**Comment by andrewgadziksonos** [comment]
@JakeChampion is there anything you need from me on this?  Happy to help out if there's anything I can do

**Comment by JakeChampion** [comment]
Hi @andrewgadziksonos, sorry for not taking any action on this, I've been away on some holidays and keeping away from any programming. I'll be looking into this pull-requests and a few others today :-)

**Pull request merged**